package/src/components/Channel/Channel.tsx

```diff
@@ -1606,8 +1606,9 @@ const ChannelWithContext = <
     watchers,
   });
 
-  const inputMessageInputContext = useCreateInputMessageInputContext({
+  const inputMessageInputContext = useCreateInputMessageInputContext<StreamChatGenerics>({
     additionalTextInputProps,
+    appSettings: null,
```

Comment:
Why is it being set to null here? I thought it's already null by default?

Reply:
Yes it is null by default. I set it null here cause I am not using the appSettings hook in this file and passing appSettings directly. This is in relation to a discussion we had earlier when I suggested having the appSettingshook in the channel component but mentioned that it would make a request for appSettings on every rerender and it would be bad for performance. I might have understood wrongly but that was what I was thinking when I did this and it worked